src/sagemaker/utils.py

```diff
@@ -88,6 +88,13 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def vpc_config_dict(subnets, security_group_ids):
```

Comment:
This function should not be in utils. Maybe in session.py where container_def is defined? https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/session.py#L917

Reply:
Please, add docstrings.

Reply:
The function is imported and used in all Estimators too. It's how they translate the `subnets` and `security_group_ids` attributes into a `vpc_config` passed to the Model or Session method. For this reason I put it in utils so it can be imported from a central module.

Would you rather see 
```
from sagemaker.session import Session, vpc_config_dict
```
or
```
from sagemaker.session import Session
from sagemaker.utils import vpc_config_dict
```

either way, yes I will add docstring

Reply:
I also considered moving to a totally new module `vpc_utils` which would include the following:

```
VPC_CONFIG_DEFAULT = {}


def vpc_config_dict(subnets, security_group_ids):
    ...


def vpc_config_unpack(vpc_config):
    ...
```

Any guiding principles for when to split off a new utility module?

Reply:
Based on https://github.com/google/styleguide/blob/gh-pages/pyguide.md#224-decision. I would prefer 

```
from sagemaker import session

session.Session ...
session.vpc_config_dict
```

I am ok if you decide to move the vpc to a separate file thought.